scripts/globals/mobskills/mijin_gakure.lua

```diff
@@ -11,7 +11,7 @@ end
 
 function onMobWeaponSkill(target, mob, skill)
     local dmgmod = 1
-    local hpmod = skill:getMobHPP() / 100
+    local hpmod = mob:getHPP() / 100
```

Comment:
its supposed to be what it was. The difference is that is the HP when the mob initially used the skill, vs whatever the hp was at that exact moment which matters when the code has to execute more than once.

Reply:
I got a spit out error everytime it hit on that line

Reply:
diagnose the cause of the error instead of replacing the function

Reply:
seems my files where caught up by my server was slightly out of date